research/object_detection/core/target_assigner.py

```diff
@@ -107,7 +108,8 @@ def assign(self,
              groundtruth_boxes,
              groundtruth_labels=None,
              unmatched_class_label=None,
-             groundtruth_weights=None):
+             groundtruth_weights=None,
+             class_predictions=None):
```

Comment:
You don't need this change

Reply:
I do because in batch_assign it by default now passes in class predictions.